pysc2/lib/renderer_human.py

```diff
@@ -676,6 +677,25 @@ def draw_overlay(self, surf):
     surf.surf.blit(text, (3, 3))
 
   @sw.decorate
+  def draw_shield_energy_status(self, surf):
```

Comment:
Could this just be merged into draw_units? It's very much related, and would let you skip the intersection check.